lgc/include/lgc/patch/PatchBufferOp.h

```diff
@@ -30,6 +30,7 @@
  */
 #pragma once
 
+#include "Utils.h"
```

Comment:
Move that to the bottom to retain the ordering of the includes (by name)?

Reply:
Also note that, in LGC, we use the LLVM style (and general open source style) of public include files being in a subdirectory which gets named in the #include, e.g. `#include "lgc/Utils.h"`.